static/js/theme-map/VerticalFullPageMapOrchestrator.js

```diff
@@ -443,11 +449,27 @@ class VerticalFullPageMapOrchestrator extends ANSWERS.Component {
       el.classList.remove('yxt-Card--pinFocused');
     });
 
-    this._detailCard && this._detailCard.remove();
+    this._removeElement(this._detailCard);
 
     this.core.storage.set(StorageKeys.LOCATOR_SELECTED_RESULT, null);
   }
 
+  /**
+   * Removes an element from the DOM, with support for IE11
+   * 
+   * @param {Element} element
+   */
+  _removeElement(element) {
```

Comment:
could this go somewhere in the "Util" folder?

Reply:
sure